### PR TITLE
Twitchshoutout

### PR DESCRIPTION
**Pull request by Conundris**
#13 
- Added Embed message 
image

### CODE REVIEW

File: app/src/database.rs [hunk 1]
```diff
@@ -242,19 +242,25 @@ impl Handle {
         data: &Member
     ) -> Result<(), ()>
     {
-        if let Err(_) = self.connection.execute(
-            "INSERT OR REPLACE INTO Members(
+        let rowschanged = match self.connection.execute("INSERT OR REPLACE INTO Members(
```
Comment (skrytt) [inline app/src/database.rs]: Minor: as a stylistic convention for multi-word variable names I'd prefer snake_case convention for easy readability

File: app/src/stream_notify.rs [hunk 2]
```diff
@@ -235,11 +235,11 @@ fn stream_notify(
...
                         .url(&stream_url) // Stream URL
                         .author(|a| {
-                            a.name(format!("{:?} {:?}", &user_title, &member_name))
+                            a.name(format!("{} {}", &user_title, &member_name))
                                 // Gets pfp url or just discords default URL for pfp
                                 .icon_url(member.user.read().face())
                         })
-                        .field("Playing", format!("{:?}", &stream_game), true) // Game being Played
+                        .field("Playing", format!("{}", &stream_game), true) // Game being Played
                         //.footer(|f| f.text(format!("Stream started at 13:37"))) // Point out stream starting time
```
Comment (skrytt) [inline app/src/stream_notify.rs]: Let's try to avoid leaving commented out code, we can always add things back in future commits

**Comment by skrytt** [comment]
@alexisnotonffire I'm happy with the change, let me know your thoughts or if I should go ahead and merge to master

File: app/src/stream_notify.rs (hunk 2 above)
Comment (Conundris) [reply app/src/stream_notify.rs]: missed that one, was paying more attention to not take any of my local env changes in this :D

**Pull request merged**